operations/iscsi_sbps/Node_Personalization_Verification.md

```diff
@@ -0,0 +1,256 @@
+# Node Personalization Verification
+
+* [Introduction](#introduction)
+* [Post-personalization Verification](#post-personalization-verification)
+     * [Check SBPS Marshal Agent status](#check-sbps-marshal-agent-status)
+     * [Check `targetcli ls` command output](#check-targetcli-output)
+     * [Check for DNS SRV and A records](#check-for-dns-srv-and-a-records)
+     * [Run readiness checks](#run-readiness-checks)
+
+## Introduction
+
+Worker node personalization installs SBPS Marshal agent, adds kubernetes labels for the worker NCNs intended to be configured as iSCSI targets, it also removes the labels from workers that are not designated to be iSCSI targets, creates DNS
+SRV and A records etc. See [Managing selective node personalization](Managing_selective_node_personalization.md) for details on how worker NCNs designated to be enabled as iSCSI targets.
+
+## Post-personalization Verification
+
+(`ncn-w#`) Run these checks on worker nodes that are enabled as an iSCSI target in order to verify that it is
```

Comment:
Why does the DNS step need to be run on every iSCSI-enabled worker? Doesn't the output from running it once show all of the DNS entries for the other workers as well?